warehouse/query-core/src/main/java/datawave/query/predicate/EventDataQueryFieldFilter.java

```diff
@@ -47,7 +60,10 @@ public EventDataQueryFieldFilter(ASTJexlScript script, Set<String> nonEventField
             queryFields.add(JexlASTHelper.deconstructIdentifier(identifier));
         }
 
-        setIncludes(queryFields);
+        // Future change, leave in for now.
+        // this.keyProjection = new KeyProjection(queryFields, Projection.ProjectionType.INCLUDES);
```

Comment:
Not a big fan of commented out code. If this is the API we want as the long-term solution, it'd be better to put that in a ticket rather than having commented code committed.

Reply:
Minimally, it should be `projection` instead of `keyProjection` but that highlights the point that the code in a comment is harder to maintain.

Reply:
This ticket is the first part of a series of six tickets. I left it in there so for the follow on tickets it can be uncommented but sure I'll rip it out.